test/solidity/Libraries/LibUtil.t.sol

```diff
@@ -41,17 +41,43 @@ contract LibUtilImplementer {
     function revertWith(bytes memory reason) public pure {
         LibUtil.revertWith(reason);
     }
+
```

Comment:
Do we benefit at all by testing through an implementor contract rather than just testing the lib directly? Adds a bit of mental overhead (though not much) in my opinion.

Reply:
Foundry fails if we do not do that. The implementer was only added a few weeks ago when the foundry tests started to fail with something like "reverts that happen directly inside the test case cannot be caught by vm.expectRevert"